src/utils/hashPassword.js

```diff
@@ -0,0 +1,13 @@
+const bcrypt = require('bcryptjs');
+
+const hashPassword = (password, callback) => {
+  bcrypt.genSalt(10, (err, salt) => {
```

Comment:
we can use `bcrypt.hash` to auto-generate salt, So try to use it without generating salt
**Documentation is your friend**